Form/DirectType.php

```diff
@@ -39,7 +39,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     /**
      * @return string
      */
-    public function getName()
+    public function getBlockPrefix()
```

Comment:
This is a BC break, we cannot get rid of `getName()` method since we still support Symfony 2.8.
Just add the `getBlockPrefix()` instead of renaming `getName()`